win-odbc64/odbcclient/drvr35/cstmt.cpp

```diff
@@ -1063,7 +1063,20 @@ SQLRETURN CStmt::SendSQLCommand(BOOL SkipProcess, SQLCHAR *StatementText,
             else
                 m_StmtType = TYPE_UNKNOWN;
         }
-
+        else if (strcmp(token, "SET") == 0)
+        {
+            token = strtok(NULL, delimiters);
+            if (token != NULL && strcmp(token, "NAMES") == 0)
+            {
+                token = strtok(NULL, delimiters);
+                if (token != NULL && strcmp(token, "'UTF8'") == 0)

```

Comment:
I'm wondering if there should be an "else" for this "if"? Suppose I say, "SET NAMES 'UTF9'", what happens?

Reply:
The buffer will be considered as a query and sent to server.